src/hooks/http.ts

```diff
@@ -346,7 +346,9 @@ export class Http {
         if (isEmptyString(requestData.body)) {
           const body = extractBodyFromEmitSocketEvent(args[1]);
           if (body) {
-            requestData.body += body.substr(0, getEventEntitySize(false));
+            const eventEntitySize = getEventEntitySize(false);
+            requestData.body += body.substr(0, eventEntitySize);
+            requestData.truncated = eventEntitySize < requestData.body.length;
```

Comment:
```suggestion
            requestData.truncated = eventEntitySize < body.length;
```
I think that you're truncate only the body

Reply:
why ask on the newBody we should ask on the whole body collected so far